src/OpenTK.Windowing/GraphicsLibraryFramework/Interfaces/IGLFW.cs

```diff
@@ -0,0 +1,19 @@
+﻿using AdvancedDLSupport;
+
+namespace OpenToolkit.Windowing.GraphicsLibraryFramework.Interfaces
+{
+    /// <summary>
+    /// Defines the public interface of the GLFW 3.2 library
+    /// </summary>
+    [NativeSymbols(Prefix = "glfw")]
+    public interface IGLFW
```

Comment:
If we're defining unsafe methods, better to make the interface internal

Reply:
No bueno
`Generated_GLFW_d4028dc4_2336_488d_ab8d_08ace22d970b' from assembly 'DLSupportDynamicAssembly, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' is attempting to implement an inaccessible interface.'`

Reply:
@daerogami Internal interfaces are supported, but you'll have to add an InternalsVisibleTo("DLSupportDynamicAssembly") attribute to the main assembly.

Reply:
Fixed.